libraries/postgres/src/Schema.js

```diff
@@ -18,7 +18,7 @@ class Schema {
    */
   static fromDbDirectory(directory) {
     const dentries = fs.readdirSync(path.join(directory, 'versions'));
-    let versions = new Array(dentries.length);
+    let versions = [];
```

Comment:
What bug is this fixing?

Reply:
I put some `.sql` files in `db/versions`, and this function got all confused since it counts the entries in the directory and assumes every one is a DB version.  The new version just extends the array as necessary for each version file it finds.